minecraft_script/lint.py

```
@@ -0,0 +1,51 @@
+    code: str,
+    *,
+    source_path: str | Path | None = None,
+    import_base_dir: str | Path | None = None,
+) -> list[Diagnostic]:
+    try:
+        parse_code_with_imports(
+            code,
+            source_path=source_path,
+            import_base_dir=import_base_dir,
+        )
+    except BaseException as error:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `BaseException`

(BLE001)